src/servicesHost.ts

```diff
@@ -852,6 +852,14 @@ export function makeSolutionBuilderHost(
   function getInputFileNameFromOutput(
     outputFileName: string
   ): string | true | undefined {
+    // Unless we explicitly want to compile files in node_modules, exclude them from lookups
+    if (
+      !instance.loaderOptions.allowTsInNodeModules &&
+      outputFileName.indexOf('node_modules') !== -1
```

Comment:
```suggestion
      outputFileName.indexOf('/node_modules/') !== -1
```

Reply:
Note: in the compiler, filenames get normalized with forward slashes, so I was assuming that here, but if it comes directly from webpack, maybe this is a bad assumption

Reply:
I had originally included slashes, but worried about path normalization. The other point in code that does TsInNodeModules checks without slashes, so I adopted that:
https://github.com/TypeStrong/ts-loader/blob/master/src/index.ts#L109

That being said, I can test on Windows and Mac to double check

Reply:
Unfortunately, it's platform sensitive
<img width="705" alt="Screen Shot 2020-12-29 at 1 08 29 PM" src="https://user-images.githubusercontent.com/1418465/103314156-feb47380-49d6-11eb-86bf-30644bfea893.png">